jac-uk/apply
# [admin#1933] Remove QT Code

## What's included?
Remove qualifying test code from Apply site.

Closes jac-uk/admin#1933

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Previously a logged in user would be able to access "Online Tests". Check that this has now been removed.

e.g. 

**Before**
<img width="303" alt="image" src="https://github.com/jac-uk/apply/assets/8524401/c83c74df-35dd-4d1a-80d7-544af94b5714">

**After**
<img width="298" alt="image" src="https://github.com/jac-uk/apply/assets/8524401/929b61e4-d590-4af0-8d68-16e21deae886">



## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
